roles/dnsmasq/molecule/default/converge.yml

```
@@ -18,6 +18,17 @@
 - name: Converge
   hosts: all
   vars:
+    cifmw_dnsmasq_enable_dns: true
+    cifmw_dnsmasq_interfaces:
+      - mol-test-1
+    cifmw_dnsmasq_addresses:
+      - fqdn: "api.ocp.openstack.lab"
+        address: "192.168.122.2"
+      - fqdn: ".apps.ocp.openstack.lab"
+        address: "192.168.122.3"
+    cifmw_dnsmasq_listen_addresses:
+      - "192.168.254.9"
+    cifmw_dnsmasq_domainname: "ocp.openstack.lab"
```

this reads better maybe?
```suggestion
    cifmw_dnsmasq_domain_name: "ocp.openstack.lab"
```